.editorconfig

```diff
@@ -0,0 +1,27 @@
+; https://editorconfig.org/
+
+root = true
+
+[*]
+insert_final_newline     = true
+charset                  = utf-8
+trim_trailing_whitespace = true
+indent_style             = space
+indent_size              = 2
```

Comment:
I'd think 4 is a better value for `indent_size`. Not too much and not too little. Easier to distinguish different indent levels.